src/components/form-elements/Textarea.jsx

```diff
@@ -17,9 +17,10 @@ export const SIZE = {
 };
 
 export type TextareaPropsType = {
+  // $FlowFixMe any generic prop types here broke autocomplete, so let's leave it as is for now
```

Comment:
this is an argument to switch to typescript

my vscode almost never proviedes autocomplete

Reply:
1) your vscode doesn't provide autocomplete because of typescript :) it just hangs on big projects such as ours and because of poor vscode architecture it doesn't show autocomplete from anything else if TS is broken. Check their github, they have tons of such issues.
2) code should be not only typed, but typechecked. It's a main purpose of type system. Typescript can't check type usage, it can check only type definitions. Like eslint, but for types. Type system based not on graphs doesn't provide any runtime guarantees, so it doesn't work by definition.